assets/airports/kstl.json

```diff
@@ -1085,7 +1085,7 @@
             "origin": "kstl",
             "destination": "",
             "category": "departure",
-            "route": "KSTL.BRAKK4.HLV",
+            "route": "KSTL30L.BRAKK4.HLV",
```

Comment:
Not sure how you're going about selecting runways, however, should this be 29?

Reply:
Truth is there isn't really a set runway for each procedure. IRL, the ground controller sort of chooses either 29 or 30L depending on the traffic. 30L is the main departure runway though and 29 is only used in heavy traffic conditions, so that's sort of what I went for; 30L for most departures and 29 for those that are clearly better suited for it. In regards to the LINDY6 and PLESS5, these are vector departures, so aircraft wouldn't actually fly to STL, they're usually given runway heading and then told to join the STL radial outbound for whatever transition they're on. I know all of this seems very arbitrary (and it is) but they don't seem to have a very good system in real life either. The user can always change the assigned runway if they want to.